### DISCUSSION TITLE
dlite.io website misses a link tot Github

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#152] [Ideas]
hi Renauld,

It is not possible to click/link  to Github from the new dlite.io  website to check the status of Dlite code. Also Platform.dlite.io forces you to create an account before entering. 
Curious developers must manually search with Google for " Github dlite" to find it. I think that wil annoy site visitors and drives them away from dLite.

Folkert

**Comment by renaudpawlak** [comment]
Sorry for late answer, but I am planing to have deep changes in the project. I want plug-n-play components to become Open Source under a very permissive licence (maybe Apache or LGPL) so that anyone can build its own components without having to conform to GPL constraints. On the other hand, the core/kernel components will be closed-source but remain free of use and deploy for many contexts. The idea is that I want to Open Source what is actually useful. If we want to build a community, the Open Source should be well targeted to the kinds of contributors that will want to contribute or to use some OS components. However, if I Open Source the whole project with no guidance or framework to contribute, it will certainly not find any audience.

So, I have a quite clear idea on how to do it, but it actually requires some reengineering work and it will take some time because I also have other projects being built with dLite. That's why I removed the github link for now (it will eventually come back) because I don't want the developers to have a false idea of the project during the transition period. In the meantime, please note that platform.dlite.io continues to evolve quite fast, but the changes will not be reflected in the project until I have finished the re-engineering work.

We can continue using the github project for discussion and issues anyways.

**Comment by HFolkertH** [comment]
Ahh. That explains why i see the dlite version number going up while nothing seems to happen on the github code. I regerarly  check github code to see if you added new features or fixed errors
